### ISSUE TITLE
LaneSegmentation/train.py文件 _epoch_train函数中使用detach目的？

### ISSUE THREAD

**Issue by ssh0072** [Ascetics/LaneSegmentation#1]
你好，看了你写的代码，我有个疑问想请教你一下：
LaneSegmentation/train.py文件 _epoch_train函数中，total_loss += loss.detach().item() 
在累加训练batch的loss时，为什么要detach，目的是什么？（因为不加也不会报错）
